src/main/java/seedu/address/logic/commands/DateCommand.java

```diff
@@ -0,0 +1,86 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Event;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a special date to an existing person in the FriendDex.
+ */
+public class DateCommand extends Command {
+
+    public static final String COMMAND_WORD = "date";
```

Comment:
Got better name? I think calling it `anniversary` would feel better other than `date`.

Reply:
Hmmm the idea is that it could be used for any kind of special annual date, not just anniversaries. Any other ideas?